cmake/UriParserConfig.cmake.in

```diff
@@ -0,0 +1,56 @@
+# Author: KangLin(kl222@126.com)
+
+# UriParser
+# --------
+#
+# Find UriParser
+#
+# Find the UriParser libraries (asound)
+#
+# ::
+#
+#   This module defines the following variables:
+#      UriParser_FOUND          - True if UriParser libraries are found
+#      UriParser_LIBRARIES      - Set when UriParser libraries is found
+#      UriParser_INCLUDE_DIRS   - where to find UriParser heard directory.
+#      UriParser_DEFINITIONS    - Set definitions
+#      UriParser_VERSION_STRING - the version of UriParser found (since CMake 2.8.8)
```

Comment:
Let's go `URIPARSER_` or `uriparser_` here, with preference for the former unless you consider that very non-CMake-style.

The comment regarding CMake 2.8.8 is a good hint to bump our requirements from 2.6 to something later, in my eyes. We can also simplify the comment then.

Reply:
Note: the variables must match desired find_package invocation. That way find_package knows correctly the appropriate variables (especially if the package was found by checking FileName_FOUND).

So when you use `find_package(UriParser)`, the file must be named UriParserConfig.cmake or uriparser-config.cmake and variables prefixed by `UriParser_`.

If you prefer lowercase style, we could rename it to uriparser-config.cmake and then (better style IMHO), use `find_package(uriparser)` and then `uriparser_` prefixes..

Reply:
markand is ok!